blocks_vertical/motion.js

```diff
@@ -138,10 +133,10 @@ Blockly.Blocks['motion_pointtowards_menu'] = {
           ]
         }
       ],
-      "colour": Blockly.Colours.motion.secondary,
-      "colourSecondary": Blockly.Colours.motion.secondary,
-      "colourTertiary": Blockly.Colours.motion.tertiary,
-      "colourQuaternary": Blockly.Colours.motion.quaternary,
+      "colour": Colours.motion.secondary,
```

Comment:
Why doesn't this use the "colours_motion" extension? If it's not worth investigating then don't investigate :P Just slightly confusing.

Reply:
I assume they were just introduced at different times before/after that extension was made. Updated them all to use it though.